### DISCUSSION TITLE
How to test Fast Data configurations

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#117] [Q&A] [answered]
Hi guys,
during the setup of all the components of the Fast Data, I can see there are a lot of configurations to setup.
One problem that I see is that there is not, that I know of, a way to test that the configurations I'm writing are correct.

Is there, a way, to test that the aggregations, the er-schema, the projection-changes configurations are correct and behave like I intend too? 
I would love a unit test like approach.

Thanks

**Comment by frengooo** [comment, marked as answer]
Hi @elmalakomar,
there is a very useful fast data test template at this link:
https://github.com/mia-platform/fast-data-low-code-test-template

It helps you to test all the fast-data flow and to be sure that your configurations work fine.

**Reply by ianFar96** [reply]
Hi @silversoul93 thank you very much for the solution, I just want to point out for the record that this repo is indeed sponsored in our Fast Data documentation https://docs.mia-platform.eu/docs/fast_data/configuration/single_views#create-the-single-view-creator-service in the Danger message :smile:

**Reply by frengooo** [reply]
Hi @ianFar96, yes, you are right.
I found it on the documentation that you linked 😄

Just a feedback, I found it but it wasn't so easy, probably a "Testing the Fast Data" section would be very useful for the user.

**Reply by ianFar96** [reply]
Noted! Thanks
